### PR TITLE
Added docs for Permanently deleting a drive item.

### PR DESCRIPTION
**Pull request by MahaGoyal**
Added docs for Permanently deleting a drive item.

### CODE REVIEW

**Comment by MahaGoyal** [comment]
ReadyForReview

**Comment by MahaGoyal** [comment]
ReadyForReview

**Comment by MahaGoyal** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by MahaGoyal** [comment]
Hi,

Yes I agree to CLA. Please find the required information below:

@microsoft-github-policy-service agree company="Microsoft"

Thanks,
Mahak
________________________________

**Comment by MahaGoyal** [comment]
ReadyForReview

**Comment by MahaGoyal** [comment]
ReadyForReview

**Comment by Danielabom** [comment]
Hi @MahaGoyal, I contacted you via Teams. Please remove the snippets from the PR, as they will be automatically generated. Thanks.

**Comment by MahaGoyal** [comment]
> Hi @MahaGoyal, I contacted you via Teams. Please remove the snippets from the PR, as they will be automatically generated. Thanks.
 Hi @Danielabom , Could you please specify which snippets you want me to remove. I havn't added anything explicitely, but please let me know if something got added.

File: changelog/Microsoft.FileServices.json [hunk 1]
```diff
@@ -1,5 +1,41 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "a8d1ea6c-3868-41c8-b499-ed81848134bd",
+          "ApiChange": "Method",
+          "ChangedApiName": "permanentDelete",
+          "ChangeType": "Addition",
+          "Description": "Added the [permanentDelete] method to the [driveItem] [DriveItem-beta] resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.FileServices.json:10]: Please follow example 7 on this wiki and update the description accordingly: [Changelog](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)

File: changelog/Microsoft.FileServices.json [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+      "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "bb7f0809-9061-4b82-b872-249661a3d329",
+          "ApiChange": "Method",
+          "ChangedApiName": "permanentDelete",
+          "ChangeType": "Addition",
+          "Description": "Added the [permanentDelete] method to the [driveItem] [DriveItem-beta] resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.FileServices.json:28]: Please follow example 7 on this wiki and update the description accordingly: [Changelog](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)

File: api-reference/v1.0/api/driveitem-permanentDelete.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+author: mahgoyal
+ms.date: 06/28/2023
+title: Permanently delete a file or folder
```
Comment (Danielabom) [inline api-reference/v1.0/api/driveitem-permanentDelete.md:4]:
```suggestion
title: Permanently delete a file or folder.
```

File: api-reference/v1.0/api/driveitem-permanentDelete.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+description: "Permanently delete a DriveItem by using its ID."
```
Comment (Danielabom) [inline api-reference/v1.0/api/driveitem-permanentDelete.md:7]:
```suggestion
description: "Permanently delete a driveItem by using its ID."
```

File: api-reference/v1.0/api/driveitem-permanentDelete.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+doc_type: apiPageType
+---
+# Permanently delete a DriveItem
```
Comment (Danielabom) [inline api-reference/v1.0/api/driveitem-permanentDelete.md:10]:
```suggestion
# Permanently delete a driveItem
```

File: api-reference/v1.0/api/driveitem-permanentDelete.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Namespace: microsoft.graph
+
+Permanently deletes a [DriveItem](../resources/driveitem.md) by using its ID.
+Note that deleting items using this method will be permanently deleted and won't go to recycleBin. Hence can't be restored.
```
Comment (Danielabom) [inline api-reference/v1.0/api/driveitem-permanentDelete.md:15]:
```suggestion
Note that if you delete items using this method, they will be permanently removed and won't be sent to the recycle bin. Therefore, they cannot be restored afterward.
```

File: api-reference/v1.0/api/driveitem-permanentDelete.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
+---
```
Comment (Danielabom) [inline api-reference/v1.0/api/driveitem-permanentDelete.md:1]: Filenames must be in lowercase.

**Review by Danielabom — CHANGES_REQUESTED** [review]
@MahaGoyal, I've added comments in the v1 files, but please apply those changes to the beta files as well. Thanks!

File: changelog/Microsoft.FileServices.json (hunk 1 above)
Comment (MahaGoyal) [reply changelog/Microsoft.FileServices.json:10]: Actually it was giving link error with format given in wiki as link generated was not present. So I removed the links. Is there any way these links can be validated ?

File: api-reference/v1.0/api/driveitem-permanentDelete.md (hunk 7 above)
Comment (MahaGoyal) [reply api-reference/v1.0/api/driveitem-permanentDelete.md]: Done. Thanks

File: api-reference/v1.0/api/driveitem-permanentDelete.md (hunk 6 above)
Comment (MahaGoyal) [reply api-reference/v1.0/api/driveitem-permanentDelete.md]: Done

File: api-reference/v1.0/api/driveitem-permanentDelete.md (hunk 5 above)
Comment (MahaGoyal) [reply api-reference/v1.0/api/driveitem-permanentDelete.md]: Done

File: api-reference/v1.0/api/driveitem-permanentDelete.md (hunk 4 above)
Comment (MahaGoyal) [reply api-reference/v1.0/api/driveitem-permanentDelete.md]: Done

**Comment by MahaGoyal** [comment]
> @MahaGoyal, I've added comments in the v1 files, but please apply those changes to the beta files as well. Thanks!

Done. Thanks.

**Comment by MahaGoyal** [comment]
#feedback-addressed

File: changelog/Microsoft.FileServices.json (hunk 1 above)
Comment (MahaGoyal) [reply changelog/Microsoft.FileServices.json:10]: Done I've made the changes, and added required links in changeLog changes.

**Comment by MahaGoyal** [comment]
It was closed by mistake.

File: changelog/Microsoft.FileServices.json [hunk 8]
```diff
@@ -1,5 +1,41 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "a8d1ea6c-3868-41c8-b499-ed81848134bd",
+          "ApiChange": "Method",
+          "ChangedApiName": "permanentDelete",
+          "ChangeType": "Addition",
+          "Description": "Added the [permanentDelete](https://learn.microsoft.com/en-us/graph/api/driveItem?view=graph-rest-beta) method to the [driveItem](https://learn.microsoft.com/en-us/graph/api/driveItem?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.FileServices.json:10]:
```suggestion
          "Description": "Added the [permanentDelete](https://learn.microsoft.com/en-us/graph/api/permanentDelete?view=graph-rest-beta) method to the [driveItem](https://learn.microsoft.com/en-us/graph/api/driveItem?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.FileServices.json [hunk 9]
```diff
@@ -1,5 +1,41 @@
...
+      "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "bb7f0809-9061-4b82-b872-249661a3d329",
+          "ApiChange": "Method",
+          "ChangedApiName": "permanentDelete",
+          "ChangeType": "Addition",
+          "Description": "Added the [permanentDelete](https://learn.microsoft.com/en-us/graph/api/driveItem?view=graph-rest-1.0) method to the [driveItem](https://learn.microsoft.com/en-us/graph/api/driveItem?view=graph-rest-1.0) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.FileServices.json:28]:
```suggestion
          "Description": "Added the [permanentDelete](https://learn.microsoft.com/en-us/graph/api/permanentDelete?view=graph-rest-1.0) method to the [driveItem](https://learn.microsoft.com/en-us/graph/api/driveItem?view=graph-rest-1.0) resource.",
```

File: api-reference/beta/api/driveitem-permanentdelete.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
+---
+author: mahgoyal
+ms.date: 06/28/2023
+title: Permanently delete a file or folder
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+description: "Permanently delete a driveItem by using its ID."
+doc_type: apiPageType
+---
+# Permanently delete a driveItem
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-permanentdelete.md:10]: The title and H1 should be driveItem: permanentDelete.

File: api-reference/beta/api/driveitem-permanentdelete.md [hunk 11]
```diff
@@ -0,0 +1,83 @@
...
+
+Namespace: microsoft.graph
+
+Permanently deletes a [DriveItem](../resources/driveitem.md) by using its ID.
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-permanentdelete.md:14]: Revise as: 
Permanently delete a [driveItem](../resources/driveitem.md) by using its ID.

File: api-reference/beta/api/driveitem-permanentdelete.md [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+## Request body
+Do not supply a request body for this method.
+
+## Response
+
+If successful, this method returns `204 No Content` response code. It does not return anything in the response body.
+
+## Example
+
+Here is an example of how to call this API.
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-permanentdelete.md:50]: Revise as: The following example shows how to call this API.

File: api-reference/beta/api/driveitem-permanentdelete.md [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+
+<!-- { "blockType": "request", "name": "permanentDelete-item", "scopes": "files.readwrite", "target": "action" } -->
+
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-permanentdelete.md:53]: Add an H3 Request.

File: api-reference/beta/api/driveitem-permanentdelete.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+<!-- { "blockType": "response" } -->
+
+```http
+HTTP/1.1 204 No Content
+```
+
+### Error responses
+
+See [Error Responses][error-response] for more info about
```
Comment (Lauragra) [inline api-reference/beta/api/driveitem-permanentdelete.md:70]: Revise as: For details about how errors are returned, see [Error responses](error-response).

File: api-reference/v1.0/api/driveitem-permanentdelete.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
+---
+author: mahgoyal
+ms.date: 06/28/2023
+title: Permanently delete a file or folder
+ms.localizationpriority: medium
+ms.prod: "sharepoint"
+description: "Permanently delete a driveItem by using its ID."
+doc_type: apiPageType
+---
+# Permanently delete a driveItem
```
Comment (Lauragra) [inline api-reference/v1.0/api/driveitem-permanentdelete.md:10]: Please apply the same fixes to the v1.0 topic.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**